app/experimenter/nimbus-ui/src/components/LinkExternal/index.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import LinkExternal from "./index";
+import { ReactComponent as OpenExternalIcon } from "./open-external.svg";
+
+storiesOf("Components/LinkExternal", module).add("basic", () => (
+  <p className="m-3">
+    <LinkExternal href="https://mozilla.org">
+      Keep the internet open and accessible to all.
+      <OpenExternalIcon
```

Comment:
Noticed the issue called for "Create a LinkExternal component for linking outside of the project and add the SVG"

But, I also noticed that LinkExternal doesn't actually itself use the SVG indicating an external link. Seems like there's [just one spot where a component *using* LinkExternal includes the icon](https://github.com/mozilla/fxa/blob/7a4afec1b5a012afc7d470bca27163b4544105ee/packages/fxa-settings/src/components/Nav/index.tsx#L61). Did we want to make that a built-in part of LinkExternal over on nimbus?

Reply:
Yeah so, I apologize for not editing the ticket well enough or explicitely calling your attention to this comment I left yesterday: https://github.com/mozilla/experimenter/issues/3460#issuecomment-705240556

The SVG docs are super helpful so thank you for porting those over, but yeah we don't want the SVG _in_ `LinkExternal` (we don't need any SVGs _yet_) since we'll use `LinkExternal` any time we need to link-out to open a new window even with inline text. I'm sort of indifferent on keeping the SVG around and showing it in Storybook as a possibility, we we or may not use that icon later?

I'm reading through my edited issues more carefully btw to try to prevent any confusion like this on other tickets. 🙈